ionic_user_interface/src/views/UserRoutes.vue

```diff
@@ -5,41 +5,55 @@
         <ion-row class="ion-align-items-center ion-justify-content-center">
           <ion-col class="ion-align-self-center" size-lg="6" size-md="8" size-xs="12">
             <div class="page-title">
-              <b>Routes By User</b>
+              <strong>Routes By User</strong>
             </div>
             <ion-item class="rounded">
               <ion-icon
                 slot="start"
                 class="ion-no-margin margin-right"
                 :icon="personCircleOutline"
               ></ion-icon>
-              <ion-text>{{ profileUsername }}</ion-text>
+              <ion-text class="username">{{ profileUsername }}</ion-text>
               <ion-button
-                v-if="!isOwnself && isLoggedIn"
+                v-if="isLoggedIn && !isOwnself && isAdmin"
                 slot="end"
                 class="margin-left"
                 color="danger"
-                @click="reportUserHandler"
+                @click="disableUserHandler"
               >
-                <ion-label>Report&nbsp;</ion-label>
-                <ion-icon :icon="flag"></ion-icon>
+                <ion-label>Disable&nbsp;</ion-label>
               </ion-button>
             </ion-item>
           </ion-col>
         </ion-row>
-        <ion-row
-          class="ion-align-items-center ion-justify-content-center"
-          v-if="isLoggedIn && !isOwnself && isAdmin"
-        >
+        <ion-row class="ion-align-items-center ion-justify-content-center">
           <ion-col class="ion-align-self-center" size-lg="6" size-md="8" size-xs="12">
-            <ion-button
-              v-if="isAdmin"
-              class="margin-left"
-              color="danger"
-              @click="disableUserHandler"
-            >
-              <ion-label>Admin Disable</ion-label>
-            </ion-button>
+            <ion-row class="ion-justify-content-center">
```

Comment:
This row/col is not aligned with the other stuff on the page
![image](https://user-images.githubusercontent.com/46164638/126266126-c3aca5fa-95dd-43cc-bb92-6c632292a387.png)